components/src/core/components/Input/Select/SelectInput.vue

```diff
@@ -21,7 +21,7 @@
     </oxd-select-text>
 
     <oxd-select-dropdown
-      v-dropdown-direction
+      v-dropdown-direction="!forcedDirection"
```

Comment:
Consider using the specific positive value for this. in js many things can be interpreted as `falsy`, so i'd rather make this very specific boolean true.

```
v-dropdown-direction="forcedDirection === true"


value should be named aptly inside the directive (eg: override, disable) and should be of boolean type
```